### PR TITLE
Grafana Dashboard Fix

### PR DESCRIPTION
**Pull request by AakashJethva09**
Updated the metrics_influx.py file with the necessary changes to fix the broken dashboard

### CODE REVIEW

File: lib/metrics_influx.py [hunk 1]
```diff
@@ -30,8 +30,8 @@ def MONITOR_MILLISECONDS(k,values):
             ncores = int(values['NumberCores'])
             nvalue = value/ncores if ncores != 0 else 0.
             rvalues += ",normvalue=%f" % (nvalue)
-                
-    return rvalues
+
+return rvalues
```
Comment (thebithead) [inline lib/metrics_influx.py]: This indentation doesn't look right.  Why did you change this?

File: lib/metrics_influx.py [hunk 2]
```diff
@@ -110,139 +110,139 @@ def v(raw):
 
 
 metrics = {
-    "Objects"   :                              MONITOR_COUNT,
```
Comment (thebithead) [inline lib/metrics_influx.py]: Does this just re-arrange the metrics into alphabetical order, or did any of the types/actions change?

File: lib/metrics_influx.py [hunk 3]
```diff
@@ -262,17 +262,17 @@ def format(values):
...
     header = ["TimeStamp","NodeType","Hostname","ProcessId","NodeId","Database","Region"]
     tags = "host=%s,nodetype=%s,pid=%s,nodeid=%s,db=%s,region=%s" % (hostname,nodetype,processid,nodeid,database,region)
 
-    with closing(StringIO()) as buffer:
-        for k in values:
-            value_formatter = metrics[k] if k in metrics else None
-            if value_formatter:
-                rvalues = metrics[k](k,values)
-                print >> buffer,"%s,%s %s %s" % (k,tags,rvalues,timestamp)
+with closing(StringIO()) as buffer:
```
Comment (thebithead) [inline lib/metrics_influx.py]: This indentation looks wrong.

File: lib/metrics_influx.py (hunk 2 above)
Comment (thebithead) [reply lib/metrics_influx.py]: Can you tell which values changed?

**Comment by butson** [comment]
Not without taking the original file, sorting the array and then diffing the files.  Of course you can do that also.

**Comment by thebithead** [comment]
Also, what is the reason for the binary .DS_Store file you are adding?  I think that is Mac specific and has nothing to do with NuoCA.
